src/assets/ba_data/python/bascenev1lib/activity/multiteamvictory.py

```diff
@@ -372,39 +377,40 @@ def on_begin(self) -> None:
         v_offs = 0.0
         tdelay += len(player_entries) * 8 * t_incr
         for _score, name, prec in player_entries:
-            tdelay -= 4 * t_incr
```

Comment:
Question: what is this `if prec.player.in_game` check for? Does it break something to show players who have left? This seems kinda opposite of the code added to show the winner even after they've left.